src/Features/Core/Portable/ImplementInterface/ImplementInterfaceGenerator.cs

```diff
@@ -28,42 +28,33 @@ private sealed partial class ImplementInterfaceGenerator
     {
         private readonly Document Document;
 
-        private readonly bool Explicitly;
-        private readonly bool Abstractly;
-        private readonly bool OnlyRemaining;
-        private readonly bool ImplementDisposePattern;
+        private readonly AbstractImplementInterfaceService Service;
 
-        private readonly ISymbol? ThroughMember;
-        private readonly ImplementTypeGenerationOptions Options;
         private readonly IImplementInterfaceInfo State;
-        private readonly AbstractImplementInterfaceService Service;
+        private readonly ImplementTypeGenerationOptions GenerationOptions;
```

Comment:
In https://github.com/dotnet/roslyn/pull/74323 I removed ImplementTypeGenerationOptions, which grouped ImplementTypeOptions with other options that are no longer needed. So this should now be ImplementTypeOptions and you'd probably want to wait for that change and rename GenerationOptions.

Reply:
yes.  will do that.

Reply:
done.